workspaces/cli-shared/package.json

```diff
@@ -40,7 +40,7 @@
     "ora": "^4.0.4",
     "proper-lockfile": "^4.1.1",
     "semver": "^7.3.4",
-    "shape-hash": "1.0.25",
+    "shape-hash": "1.0.6",
```

Comment:
why back to 1.0.6?

Reply:
wip. there will be a PR an insourced one inside the repo.